AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -749,13 +759,13 @@ - (NSMutableDictionary *)getReportingParametersForUpdatedRelease:(NSString *)upd
 
   // Check if we need to report release installation.
   NSString *lastDownloadedReleaseHash = [MS_USER_DEFAULTS objectForKey:kMSDownloadedReleaseHashKey];
-  if (!lastDownloadedReleaseHash) {
+  if (lastDownloadedReleaseHash == nil) {
```

Comment:
I like that. `!` is so easy to miss.

Reply:
I like this one better. It is a trade-off but it will reduce one bit operation.